pom.xml

```diff
@@ -4,15 +4,15 @@
 
   <groupId>vg.civcraft.mc.civmodcore</groupId>
   <artifactId>CivModCore</artifactId>
-  <version>1.5.0</version>
+  <version>1.5.01</version>
 
   <name>CivModCore</name>
   <url>https://github.com/Civcraft/CivModCore/</url>
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <maven.compiler.source>1.7</maven.compiler.source>
-    <maven.compiler.target>1.7</maven.compiler.target>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
Are we sure we want to upgrade Java versions? Wouldn't this force all plugins that use CivModCore to use Java 8?